test/tstDetailsAlgorithms.cpp

```diff
@@ -14,159 +14,153 @@
 
 #define BOOST_TEST_MODULE DetailsAlgorithms
 
-namespace details = ArborX::Details;
+// NOTE: message is not required any more with C++17
+#define STATIC_ASSERT(cond) static_assert(cond, "");
```

Comment:
I would argue that defining a macro is more confusing than typing the extra four characters, but since it's just a test...

Reply:
I had called the initial version of the macro `BOOST_TEST_AND_STATIC_ASSERT` because I thought I remembered a Boost.Test issue about unit tests with no assertion somehow reporting failure.  But return code was still zero when I dropped the `BOOST_TEST`.